src/CRUD.js

```diff
@@ -0,0 +1,87 @@
+export const addItem = (arr, value) => {
+  if (JSON.parse(localStorage.getItem('todos')) == null) {
+    arr = [];
+  } else {
+    arr = JSON.parse(localStorage.getItem('todos'));
+  }
+
+  const length = arr.length + 1;
+  const obj = {
+    description: value,
+    completed: false,
+    index: length,
+  };
+  arr.push(obj);
+  localStorage.setItem('todos', JSON.stringify(arr));
+};
+
+export const removeItem = (id) => {
+  let arr = JSON.parse(localStorage.getItem('todos'));
+  arr = arr.filter((e) => e.index.toString() !== id.toString());
+  for (let i = 0; i < arr.length; i += 1) {
+    arr[i].index = i + 1;
+  }
+  localStorage.setItem('todos', JSON.stringify(arr));
+};
+
+const updateItem = (value, id) => {
+  const arr = JSON.parse(localStorage.getItem('todos'));
+  arr[id - 1].description = value.trim();
+  localStorage.setItem('todos', JSON.stringify(arr));
+};
+
+export const getDescriptionInput = (input, arr, id) => {
+  const inputDescription = input;
+  input.addEventListener('keyup', () => {
+    // const valLen = inputDescription.value.length;
+    // if (valLen > 0) {
+    updateItem(inputDescription.value, id);
+    // } else {
+    //   setTimeout(() => {
+    //     inputDescription.value = arr[id - 1].description;
+    //   }, 2000);
+    // }
+  });
+};
```

Comment:
- [ ] Consider using a `submit` event because the `keyup` can send multiple useless requests
- [ ] Consider Removing the comment before pushing on github